### DISCUSSION TITLE
How to set `publicPath` in defineConfig in version 1.38.0?

### DISCUSSION THREAD

**Discussion by Michael18811380328** [web-infra-dev/rspress#1680] [General]
I use several methods to set publicPath in defineConfig, but it can not work(outDir or publicPath is not work), so how to solve this issue? I just want to put doc_build to another path.

**Comment by Timeless0911** [comment]
Set `builderConfig.output.assetPrefix`

**Reply by Michael18811380328** [reply]
Thx! I will try this config.
